housekeeper/store/models.py

```diff
@@ -63,7 +64,9 @@ class Version(Model):
     archive_checksum = Column(types.String(256), unique=True)
 
     bundle_id = Column(ForeignKey(Bundle.id, ondelete="CASCADE"), nullable=False)
-    files = orm.relationship("File", backref="version", cascade="delete, save-update")
+    files = orm.relationship(
+        "File", backref="version", cascade="delete, save-update", cascade_backrefs=False
```

Comment:
Isn't the `save-update` option being "overwritten" by the `cascade_backrefs=False`? My question is what would happen if you omit the `save-update`?

Reply:
No, not over written. But `cascade_backrefs` restricts `save-update` to only go one way for a relationship (from parent to children and not the other way). So omitting it would mean changing the behaviour in a way which is out of scope for this PR

Reply:
I guess it has been done inconsistently, below it was set on the `backref(...)` instead. I'll look it over to see if it matters 👍

Reply:
Hm, the old warnings come back when set on the backref. Leaving it as is now.